src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/PublicationSummary.test.tsx

```diff
@@ -0,0 +1,332 @@
+import { render, screen, within } from '@testing-library/react';
+import React from 'react';
+import { noop } from 'lodash';
+import PublicationSummary from '@admin/pages/admin-dashboard/components/PublicationSummary';
+import {
+  MyPublication,
+  MyPublicationMethodology,
+  PublicationContactDetails,
+} from '@admin/services/publicationService';
+import { MemoryRouter } from 'react-router-dom';
+import { MyRelease } from '@admin/services/releaseService';
+
+describe('PublicationSummary', () => {
+  const testTopicId = 'test-topic';
+
+  const testContact: PublicationContactDetails = {
+    id: 'contact-1',
+    contactName: 'John Smith',
+    contactTelNo: '0777777777',
+    teamEmail: 'john.smith@test.com',
+    teamName: 'Team Smith',
+  };
+
+  const fullPermissions: MyPublication['permissions'] = {
+    canCreateReleases: true,
+    canUpdatePublication: true,
+    canCreateMethodologies: true,
+    canManageExternalMethodology: true,
+  };
+
+  const testPublication: MyPublication = {
+    id: 'publication-1',
+    title: 'Publication 1',
+    contact: undefined,
+    releases: [],
+    methodologies: [],
+    permissions: {
+      canCreateReleases: false,
+      canUpdatePublication: false,
+      canCreateMethodologies: false,
+      canManageExternalMethodology: false,
+    },
+  };
+
+  const testReleases: MyRelease[] = [
+    {
+      id: 'rel-1',
+      latestRelease: true,
+      published: '2021-06-30T11:21:17.7585345',
```

Comment:
Would suggest stripping out anything more precise than seconds as it's just noise otherwise - same for the other releases

Reply:
Ditto